scripts/l4y3rc4k3/path.gv.jinja

```diff
@@ -0,0 +1,51 @@
+digraph {{ branch_id }} {
+  //splines = ortho;
+  //concentrate = true;
+
+  rankdir="LR";
+
+  node [shape="box", style="rounded,filled", fillcolor="red", penwidth = 2];
```

Comment:
I think fillcolor="white" would be better

Reply:
This was just to further distinguish recommended labs from optional ones.  If you would rather not, `fillcolor` settings can be changed or removed in `path.gv.jinja`, the template file from which the Python script generates graphs for the course and optional labs with multiple components.

Reply:
How does one run the script?

Reply:
Right now the script just generates the graphs in `/misc`. I haven't tried it on any other workspace, but you should be able to do something like:

```
> python3 -m pip install poetry
> cd cryptol-course/scripts/l4y3rc4k3
> poetry install
> poetry shell
> python3 l4y3rc4k3/l4y3rc4k3.py
```

I'll add instructions for this and other scripts when they're more stable.  (And yes, rename `l4y3rc4k3` to something less silly.)

Reply:
Works!

That's very nice so far.